### PR TITLE
Release 2.3.0 and mender-convert 2.0.0 into 2.3.x

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: 08.Downloads/docs.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Mender provides images based on the following distributions:
 | Raspberry Pi 3   | Raspbian Buster Lite | [raspbian-buster-lite-mender.img.xz][raspbian-buster-lite-mender.img.xz] |
 
 <!--AUTOVERSION: "mender-%.img.xz"/mender -->
-[raspbian-buster-lite-mender.img.xz]: https://d4o6e0uccgv40.cloudfront.net/2020-02-05-raspbian-buster-lite/arm/2020-02-05-raspbian-buster-lite-mender-2.2.0b1.img.xz
+[raspbian-buster-lite-mender.img.xz]: https://d4o6e0uccgv40.cloudfront.net/2020-02-05-raspbian-buster-lite/arm/2020-02-05-raspbian-buster-lite-mender-2.2.0.img.xz
```
Comment (lluiscampos) [inline 08.Downloads/docs.md]: ping @kacf 

This link will be broken until I do `mender-convert` release (so I will not merge this PR just yet).

Before mender-convert v2 we didn't have this chicken-and-egg problem between the two release processes: is there something we can do to break this dependency? Could we make the `autoversion` tool aware of this somehow?

Comment (kacf) [reply 08.Downloads/docs.md]: Good point. The only way I can think of is to have a tag like `mender-convert-client` or something like that. The same version as simply `mender`, but applied only when doing a mender-convert update.

File: 202.Release-information/01.Release-notes-changelog/docs.md [hunk 2]
```diff
@@ -4,279 +4,52 @@ taxonomy:
...
 | Eystein Måløy Stenberg | 419 (1.1%) |
 
 | Top changeset contributors by employer | |
 |---|---|
-| Northern.tech | 718 (85.7%) |
-| RnDity | 112 (13.4%) |
+| Northern.tech | 678 (82.0%) |
+| RnDity | 86 (10.4%) |
+| fabio@tranchitella.eu | 52 (6.3%) |
+| 28045858+0lmi@users.noreply.github.com | 3 (0.4%) |
```
Comment (kacf) [inline 202.Release-information/01.Release-notes-changelog/docs.md]: You should probably cherry-pick https://github.com/mendersoftware/integration/commit/4a9ef06737643cde89a420b3e1e135e2e5996b37 and redo this one.

Comment (lluiscampos) [reply 202.Release-information/01.Release-notes-changelog/docs.md]: Thanks, fixed!

The commit is cherry-picked, I just was in the wrong branch (2.2.x) when generating the stats for 2.3.0.

**Comment by lluiscampos** [comment]
The last two commits add `mender-convert` 2.0.0 release. Please have a look at them @kacf

**Pull request merged**